client/my-sites/checkout/checkout-thank-you/redesign-v2/products/google-workspace-product.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import ThankYouProduct from 'calypso/components/thank-you-v2/product';
+import { getEmailManagementPath } from 'calypso/my-sites/email/paths';
+
+export type ThankYouGoogleWorkspaceProductProps = {
+	productFamily: string;
+	numberOfMailboxesPurchased?: number;
+	domainName: string;
+	siteSlug?: string;
+};
+
+export function ThankYouGoogleWorkspaceProduct( {
+	productFamily,
+	domainName,
+	numberOfMailboxesPurchased,
+	siteSlug,
+}: ThankYouGoogleWorkspaceProductProps ) {
+	const translate = useTranslate();
+	const emailManagementPath = getEmailManagementPath( siteSlug, domainName );
+
+	let details;
+	if ( numberOfMailboxesPurchased && numberOfMailboxesPurchased > 1 ) {
+		details = translate( '%(quantity)s mailboxes for %(domainName)s', {
+			args: { quantity: numberOfMailboxesPurchased, domainName },
+		} );
+	} else {
+		details = translate( 'For %(domainName)s', {
+			args: { domainName },
+		} );
+	}
+
+	const actions = [
+		<Button variant="primary" href={ emailManagementPath } key="manage-email">
+			{ translate( 'Manage email' ) }
+		</Button>,
+	];
+
+	return (
+		<ThankYouProduct
+			name={ productFamily } // can be either "G Suite" or "Google Workspace"',
```

Comment:
```suggestion
			name={ productFamily },
```
I would remove this comment. Technically, all G Suite subscriptions should have been migrated to Google Workspace. There might be a few odd cases where the subscription could not be migrated, but we don't actively support G Suite, so I'd avoid mentioning it to confuse developers from thinking this is an active product.